c/composer/composer_1.10.6_1.10.19_1.10.20_rhel_8.4.sh

```diff
@@ -0,0 +1,50 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: composer
+# Version	: 1.10.6, 1.10.19, 1.10.20
```

Comment:
As per line 18, build script is created for version 1.10.19. Please update accordingly

Reply:
The script is tested and working fine on rhel 8.4. Also parameterized the version in line 18.